migrate/3.0-restore.sh

```diff
@@ -110,6 +110,9 @@ function circleci_database_import() {
     then
         echo "...scaling application deployments to 0..."
         scale_deployments 0
+
+        # delete postgres to clear any remaining connections
+        kubectl delete pod -l app.kubernetes.io/name=postgresql -n "$NAMESPACE"
```

Comment:
We should wrap this in a conditional. If PG is externalized, the customer would skip it during the migration, meaning we wouldn't need to delete the pod, it's already missing.

Reply:
great catch